app/routers/sessions.py

```diff
@@ -131,6 +154,7 @@ async def update_session(session_id: str, session_updates: UpdateSession):
     * end button is clicked (end-quiz event)
     * dummy event logic added for JNV -- will be removed!
     """
+    logger.info(f"Updating session with id {session_id}")
```

Comment:
i think we can include what `session_updates` we received too. 
`new_event = jsonable_encoder(session_updates)["event"]`

Reply:
it will always be one of dummy / start / resume / end